xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -0,0 +1,239 @@
+     loaded from a special reserved FAT16 partition on the MicroSD card every
+     time that the machine is powered on. There is also no battery-backed clock
+     or freely-modifiable "CMOS" memory to hold configuration information.
+    </para>    
+    <para>
+     This means there are no conventional settings to adjust for boot device
+     order, time and date, and so on.
+    </para>
+    <para>
+     Instead, the &rpi; has a very small amount of One-Time Programmable Memory
+     (OTP memory) which can be used to configure some settings, such as whether
+     the machine should attempt to boot from USB devices or over Ethernet. It is
+     only possible to program this by preparing a bootable MicroSD card and
+     placing certain configuration files on it. 
+    </para>
+    <para>
+     This OTP memory is described on the &rpi; Foundation web site, here:
+     <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/otpbits.mdd"/>
+    </para>
+    <note>
+     <title>Time Synchronization</title>
+     <para>
+      The lack of a Real Time Clock (RTC) also means that it is essential for
+      &rpi; devices to be configured to fetch the time from a network server by
+      &ntp;.
+    </para>
+    </note>
+   </listitem>
+   <listitem>
+    <para>
+     The primary processor on the &rpi; is the GPU, not the ARM processor. It is
+     the GPU which initialises the system and loads the operating system. As a
+     result, most &rpi; operating systems do not use a bootloader. 
+    </para>  
+    <para>
+     &sle; for the &rpi; is configured with a bootloader called
+     <literal>Das U-Boot</literal>, which is loaded before the operating system
+     and can optionally show a boot menu, like GRUB on &x86; hardware.
+    </para>
+   </listitem>
+  </orderedlist>
+  
+  <para>
+   Installation can be performed from a bootable USB device, such as a memory
+   stick or USB optical drive. The exact procedure may differ from model to
+   model, depending on how the individual device is configured.
+  </para>
+  <para>
+   The &rpi; 3 Model B+ is shipped pre-configured to boot from USB devices and
+   no additional steps should be required.
+  </para>
+  <para>
+   Depending on whether this has previously been done on a particular unit, the
+   older &rpi; 3 Model B may require additional configuration to boot from USB.
+   The procedure is described on the &rpi; Foundation web site, here:
+   <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/msd.md"/>
+  </para>
+  
+ </sect2>
+ <sect2>
+  <title>
+   Network Booting the &rpi;
+  </title>
+  <para>
+   Due to the hardware's lack of on-board firmware, network-booting the &rpi;
+   using PXE is more complex than with more conventional computers.   
+  </para>
+  <para>
+   At the time of writing, network-booting &sle; on &rpi; hardware still has the 
+   status of work in progress.
```

Comment:
I would not say so, it should work with maintenance updates installed, it's just undocumented.

Reply:
Can we point to any description of how it works anywhere?

Reply:
I have added a link to the SLE docs on PXE net-booting and removed the line about WIP. That is all the info I have for now so I think it must suffice.